javagen/src/main/java/com/azure/autorest/template/ClientMethodTemplate.java

```diff
@@ -1526,6 +1526,7 @@ private String getPollingStrategy(ClientMethod clientMethod, String contextParam
             .replace("{httpPipeline}", clientMethod.getClientReference() + ".getHttpPipeline()")
             .replace("{endpoint}", endpoint)
             .replace("{context}", contextParam)
+            .replace("{serviceVersion}", clientMethod.getClientReference() + ".getServiceVersion().getVersion()")
```

Comment:
Please test the LRO case in vanilla
vanilla-tests/swagger/lro.md (remove the polling)

Reply:
you can modify the md to set polling to one or two API, that would leave others as test of default polling